Detectors/MUON/MCH/Workflow/src/TBDigitsFileReader.h

```diff
@@ -0,0 +1,93 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_MCH_TBDIGITSFILEREADER_H_
+#define O2_MCH_TBDIGITSFILEREADER_H_
+
+#include <memory>
+#include <fstream>
+
+#include "MCHBase/Digit.h"
+
+using namespace o2::mch;
+using namespace std;
```

Comment:
same comment about using namespace in headers

Reply:
Removed.